README.md

```diff
@@ -51,20 +62,23 @@ This SDK is being actively developed and is in Beta release status. At this time
 * Suggestions for future development;
 * Any other comments or feedback on this new direction.
 
-### Feature roadmap
-
-| Feature | Status |
-| ------- | ------ |
-| Login and logout via web redirect | ✅ |
-| Credential management | ✅ |
-| Secure token storage | ✅ |
-| Native SSO / Token Exchange Flow | ✅ |
-| Device Authorization Grant Flow | ✅ |
-| Resource Owner Flow | ✅ |
-| Automatically authorized URLSessionConfiguration for authorizing requests | ✅ |
-| Time synchronization via NTP | 🚧 |
-| Advanced logging and tracing | 🚧 |
-| Transparent upgrades from okta-oidc-ios | 🚧 |
+### Key Features
+
+Several key features and capabilities are introduced with this library, with some notable improvements listed below.
```

Comment:
Improvements over what? Hopefully new users will just use this instead of looking at the old SDK, yes?

Reply:
Yes, this section will be refined / replaced later, once we get closer to a 1.0 release. This README, for now, is largely targeting our prospective beta users.  Once this is released fully, we'll update the other SDK's READMEs to point here, and describe why they might want to use our new SDK.